lib/whitehall/asset_manager_storage.rb

```diff
@@ -44,5 +44,17 @@ def path
     def content_type
       MIME::Types.type_for(filename).first.to_s
     end
+
+    def size
+      response = Services.asset_manager.whitehall_asset(path)
+      if response.has_key?('size') && !response['size'].nil?
+        Integer(response['size'])
+      else
+        false
+      end
```

Comment:
I wonder whether this could all be a lot simpler if we aim to return `0` if we can't obtain the size. Then I think this whole `if`/`else` block could be replaced with `response['size'].to_i`.

Reply:
I think that's probably a much better idea.